packages/mdc-select/bottom-line/README.md

```diff
@@ -0,0 +1,99 @@
+<!--docs:
+title: "Select Label"
+layout: detail
+section: components
+iconId: menu
+path: /catalog/input-controls/select-menus/
+-->
+
+# Select Label
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/select.html">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/selects.png" width="376" alt="Select screenshot">
+  </a>
+</div>-->
+
+Select bottom line animates activation and deactivation state as the select focuses and blurs. Every select should have a bottom line.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/text-fields.html">Material Design guidelines: Text Fields</a>
+  </li>
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/menus.html">Material Design guidelines: Menus</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/select.html">Demo</a>
+  </li>
+</ul>
+
+## Usage
+
+### HTML Structure
+
+```html
+<div class="mdc-select-bottom-line"></div>
+```
+
+### Usage within `mdc-select`
+
+```html
+<div class="mdc-select" role="listbox">
+  <div class="mdc-select__surface" tabindex="0">
+    <div class="mdc-select__label">Pick a Food Group</div>
+    <div class="mdc-select__selected-text"></div>
+    <div class="mdc-select__bottom-line"></div>
```

Comment:
`mdc-select-bottom-line`

Reply:
I think `__bottom-line` is correct while this is still within this package. e.g. text field had `mdc-text-field__bottom-line` until it was eventually split to a separate package.